build.gradle

```diff
@@ -10,7 +10,7 @@ plugins {
 }
 
 group = "uk.nhs.hee.trainee.details"
-version = "0.21.3"
+version = "0.21.4"
```

Comment:
Any reason this one is only a bugfix bump, whereas the others are a minor version / feature addition?

Reply:
First one I did out of the three and I got it wrong... Will push another commit to bump minor version